pages/common/redshift.md

```diff
@@ -0,0 +1,27 @@
+# redshift
+
+> Adjust the color temperature of your screen according to your surroundings.
+
+- Turn on Redshift with 5700K temperature during day and 3600K at night:
+
+`redshift -t {{5700}}:{{3600}}`
+
+- Turn on Redshift with a manually-specified custom location:
+
+`redshift -l {{latitude}}:{{longitude}}`
+
+- Turn on Redshift with 70% screen brightness during day and 40% brightness at night:
+
+`redshift -b {{0.7}}:{{0.4}}`
+
+- Turn on Redshift with custom gamma levels (between 0 and 1):
+
+`redshift -g {{red}}:{{green}}:{{blue}}`
+
+- Turn on Redshift with temperature transitions disabled:
+
+`redshift -r`
```

Comment:
And to be clear, I listed all of the flags in separate commands, but you can actually combine all of them (well, using the `-O` flag while also having separate night/day settings would be a bit pointless). And also, pretty much every flag that accepts multiple colon-delimited parameters, will also accept a single parameter (i.e. instead of `-b 0.7:0.4` for night/day brightness, you could do `-b 0.7` for persistent brightness. same basic deal for `-g` and `-t`) -- perhaps this should be conveyed in the tldr page somehow (couldn't think of a good way when I was writing it).